tools/benchmark/src/Configuration.ts

```diff
@@ -422,7 +422,7 @@ export function benchmarkArgumentsIsCustom(
 	const isAsync = intersection.benchmarkFnAsync !== undefined;
 	const isCustom = intersection.benchmarkFnCustom !== undefined;
 	assert(
-		[isSync, isAsync, isCustom].filter((x) => x).length === 1,
+		[isSync, isAsync, isCustom].filter(Boolean).length === 1,
```

Comment:
Nit, don't need to fix. Took me a second to get this one, I'm not sure I like it.

Reply:
Yeah, I'm not sure I like it either. I would be in favor of disabling this rule. I'll disable locally for now.